lib/fathead/mdn_css/parse.pl

```diff
@@ -145,6 +145,14 @@
         $code =~ s/\r?\n/\\n/g;
     }
     $hs->eof;
+    my $initial_value;
+
+    #initial value is found in the table properties
+    my $table_properties = $dom->at('table.properties');
+    if ($table_properties) {
+        $initial_value = parse_initial_value($table_properties);
+        $description = "$description $initial_value" if $initial_value;
```

Comment:
@hchienjo it looks like this is creating problems because we're nesting `<p>` tags.

We need to append the `$initial_value` after we've wrapped the `$description` in `<p>` so the end result is:

```html
<p>$description</p>
<p>$initial_value</p>
<pre><code>$code</code></pre>
```

We don't need any `<br>` as the `<p>` and `<pre>` have padding/margins 👍